tests/regression.rs

```diff
@@ -792,19 +792,6 @@ fn test_hough_line_detection() {
     compare_to_truth_image(&lines_image, "hough_lines.png");
 }
 
-#[test]
-fn test_bilateral_filter() {
-    fn filter(image: &GrayImage) -> GrayImage {
-        let sigma_color: f32 = 20.;
-        let sigma_spatial: f32 = 2.;
-        let radius: f32 = 3. * sigma_spatial;
-        let win_size = (radius * 2. + 1.) as u32;
-        bilateral_filter(image, win_size, sigma_color, sigma_spatial)
-    }
-
-    compare_to_truth_with_tolerance("lumaphant.png", "lumaphant_bilateral.png", filter, 1)
```

Comment:
I'd like a regression test for this function.

Running `REGENERATE=1 cargo test` will regenerate the truth images.

Reply:
Ah ok the new implementation does have so pixel rounding differences so I was worried about exact compatibility but I didn't realize I could regenerate the ground truths.